optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-backend/src/test/java/org/optaplanner/workbench/screens/domaineditor/backend/server/validation/PlanningSolutionScoreHolderSaveValidatorTest.java

```diff
@@ -169,25 +169,9 @@ public void dataObjectNull() {
 
     @Test
     public void scoreHolderTypeNotRecognized() {
-        Path dataObjectPath = PathFactory.newPath("Test.java",
-                                                  "file:///dataObjects");
-        when(ioService.readAllString(Paths.convert(dataObjectPath))).thenReturn("testResult");
-
         DataObject originalDataObject = createDataObject(HardSoftScore.class);
         GenerationResult generationResult = new GenerationResult();
         generationResult.setDataObject(originalDataObject);
-        when(dataModelerService.loadDataObject(any(),
-                                               anyString(),
-                                               any())).thenReturn(generationResult);
-        when(scoreHolderUtils.extractScoreTypeFqn(originalDataObject)).thenReturn(HardSoftScore.class.getName());
-        when(scoreHolderUtils.getScoreHolderTypeFqn(HardSoftScore.class.getName())).thenReturn(HardSoftScoreHolder.class.getName());
-
-        when(assetsUsageService.getAssetUsages(HardSoftScoreHolder.class.getName(),
-                                               ResourceType.JAVA, dataObjectPath)).thenReturn(Arrays.asList(mock(Path.class)));
-
-        DataObject updatedDataObject = createDataObject(SimpleScore.class);
-        when(scoreHolderUtils.extractScoreTypeFqn(updatedDataObject)).thenReturn("UnknownScoreClassName");
-        when(scoreHolderUtils.getScoreHolderTypeFqn("UnknownScoreClassName")).thenReturn(null);
```

Comment:
What is the reason for removing a major part of the test?

Reply:
Hi @yurloc. This part is an unecessary stubbing, and since we upgraded Mockito, it started to complain. Unnecessary stubbings are mocks that do not impact the tests results. Mockito 3 now has the capability of identifying those scenarios and reporting them. What we're deleting is pretty much unused code.

Reply:
@tiagobento Thanks for explanation. I forgot to also build the related build-bootstrap PR that upgrades Mockito, so I couldn't see the problem locally.

@caponetto The test is incomplete. It does a lot of setup but an assertion is missing at the end of the test. It's the original author's mistake, not yours. You can fix the test by adding this at the end instead of deleting the unnecessary setup:
```java
        Collection<ValidationMessage> result = validator.validate(dataObjectPath,
                updatedDataObject);
        assertEquals(1,
                result.size());

        ValidationMessage message = result.iterator().next();
        assertTrue(message instanceof ScoreHolderGlobalTypeNotRecognizedMessage);
```
Adding this will make all of the existing stubbing "necessary" and the test will pass with Mockito 3.

Consider this optional. I know it's already a lot of juggling with these cross-repo PRs.

Reply:
Thanks @yurloc! Please see https://github.com/kiegroup/optaplanner-wb/pull/384/commits/420108e5e40c9998f8d2c0e493852f907b0ee99b.